docs/develop/module-specifications/spec-distribution.mdx

```diff
@@ -30,7 +30,7 @@ Collected rewards are pooled globally and distrubuted to validators and delegato
 
 All globally tracked parameters for distribution are stored within
 `FeePool`. Rewards are collected and added to the reward pool and
-distributed to validators/delegators from here.
+distributed to validators/delegators from there.
```

Comment:
Maybe we can make this sentence clearer:

Fees are collected and stored in the `FeePool` until they can be distributed to validators and their delegators.